recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -105,15 +136,16 @@ this.PreferenceExperiments = {
 
   /**
    * Start a new preference experiment.
-   * @param  {string} experimentName
-   * @param  {string} branch
-   * @param  {string} preferenceName
-   * @param  {string|integer|boolean} preferenceValue
+   * @param {string} experimentName
+   * @param {string} branch
+   * @param {string} preferenceName
+   * @param {string|integer|boolean} preferenceValue
+   * @param {PreferenceBranchType} preferenceBranchType
    * @rejects {Error}
    *   If an experiment with the given name already exists, or if an experiment
    *   for the given preference is active.
    */
-  async start(experimentName, branch, preferenceName, preferenceValue) {
+  async start(experimentName, branch, preferenceName, preferenceValue, preferenceBranchType) {
```

Comment:
Nah, that's usually the right thing. I often wish JS had keyword arguments in the same way Python does.